### PR TITLE
Added Read me; added info related to Fill 4634

### PR DESCRIPTION
**Pull request by gkrintir**

### CODE REVIEW

File: 4634_Configs/vdmDriverII_Config_PCC_4634.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+"makeScanFileConfig":
+	{
+	"InputCentralPath": "/afs/cern.ch/user/g/gkrintir/tempeos/cms/store/group/comm_luminosity/VdM/scanFill4634_151119/central2/localdata/central18/",
+	"InputDIPFile" : "/afs/cern.ch/user/g/gkrintir/tempeos/cms/store/group/comm_luminosity/VdM/scanFill4634_151119/vdm/vdm_Fill4634.csv",
+	"ScanNames" : ["X1","Y1","Y2","X2","Y3","X3","IY1","IX1","IY2","IX2"],
+	"Comment" : "Since X,Y scans are automatically differentiated, a time window can include an X and a Y scan, will still work, timestamp in UTC", 
+	"ScanTimeWindows" :[[1447946522, 1447947608],[1447947870, 1447948900],[1447949003, 1447950033],[1447950098, 1447951128],[1447963841, 1447964820],[1447964893, 1447965872],[1447966215,1447966762],[1447966840, 1447967392],[1447967567, 1447968120],[1447968195, 1447968743]],
+	"Comment" : "A few parameters that are not in our current DIP file, but should be available eventually",
+	"Comment" : "betaStar in cm, angle in microrad",
+	"BetaStar" : 400.0,
```
Comment (capalmer85) [inline 4634_Configs/vdmDriverII_Config_PCC_4634.json]: Please change the comment above to "betaStar in m" and following line to
"BetaStar"  :  4.0,

File: 4634_Configs/vdmDriverII_Config_PCC_4634.json [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	"InputPCCFiles": ["/afs/cern.ch/user/g/gkrintir/public/ForMassimo/ZeroBias2.root"],
+        "Comment": "BCIDs on which trigger read out PCC values",
+        "PCC_BCID": [644,1215,2269,2389,2589],
+        "AnalysisDir": "../Fill4634_Nov192015",
+        "Comment": "The following directories are relative to <AnalysisDir>",
+        "InputScanFile" : "cond/Scan_4634.pkl",
+        "PCCTreeName": "pccminitree",
+        "OutputSubDir" : "LuminometerData",
+        "addScanInfo": true
+	},
```
Comment (capalmer85) [inline 4634_Configs/vdmDriverII_Config_PCC_4634.json:59]: Does this work?

File: README.md [hunk 3]
```diff
@@ -1,29 +1,164 @@
...
+`python Scripts/summarizeXSEC.py Fill4634_Nov192015/PCC/results/noCorr/LumiCalibration_PCC_DGConst_4634.pkl`
+
+and the plots will be stored under the VdMFramework/ directory as 
+```
+xsecs_PCC.png, xsecsPerBCID_PCC.png, and xsecsPerScan_PCC.png
+```
+along with their .C source files for further editing, if needed.
+
+
+
```
Comment (capalmer85) [inline README.md:164]: Excellent instructions.  As Massimo uses these, he should edit for clarification.

File: vdmDriverII.py [hunk 4]
```diff
@@ -119,7 +120,7 @@
     table = {}
     csvtable = []
 
-    table, csvtable = doMakeRateFile(makeRateFileConfig)
+    table, csvtable = doMakePCCRateFile(makeRateFileConfig)
```
Comment (capalmer85) [inline vdmDriverII.py]: Ah... I see how you made it work.  :)  You can't do this.  Most other luminometers use the regular function.  You can make a if/else statemate:

if Luminometer =="PCC":
    table, csvtable = doMakePCCRateFile(makeRateFileConfig)
else:
    table, csvtable = doMakeRateFile(makeRateFileConfig)

**Comment by capalmer85** [comment]
@gkrintir This looks very nice.  There are a few comments/changes to be made.  Once you do those, I'll merge.

File: 4634_Configs/vdmDriverII_Config_PCC_4634.json (hunk 2 above)
Comment (gkrintir) [inline 4634_Configs/vdmDriverII_Config_PCC_4634.json:59]: Hi Chris, is this related to your following comment about doMakeRateFile vs doMakePCCRateFile ? :D

Comment (capalmer85) [inline 4634_Configs/vdmDriverII_Config_PCC_4634.json:59]: Yes.

**Comment by gkrintir** [comment]
Hi Chris, you can have a look at the latest commit. ;D

**Comment by capalmer85** [comment]
@gkrintir Thanks!

**Pull request merged**